pydash/pydash_app/dashboard/endpoint.py

```diff
@@ -43,9 +47,16 @@ def remove_call(self, call):
         :param call: The endpoint call to remove.
         """
         try:
-            self.calls.remove(call)
+            self._calls.remove(call)
         except ValueError:
             raise
```

Comment:
This might already have been in Tom's request, but I am a bit confused what happens here: We catch only a ValueError to pass it on unaltered? Am I missing something?

Reply:
This was already there in my request. Essentially it is me not knowing whether a ValueError that is raised within the remove call will be raised without explicitly being told to do so, which now seems obvious that it does. This try-except structure is also used in `dashboard.py`. Will be cleaned up momentarily.